### ISSUE TITLE
how to disable shortcuts?

### ISSUE THREAD

**Issue by gittyup2018** [G33kDude/Neutron.ahk#26]
Hi, 

IE has a lot of built-in shortcuts and I would like to disable all of them but not sure how to add multiple keys? can you show me an example with multiple keys please? 

I know that I also have to define the virtual keycode from Win32 API 

VK_F5 := 0x74 but I am confused about this code that looks like JSON:

disabled_shortcuts :=
	( Join ; ahk
	{
		0: {
			this.VK_F5: true
		},
		this.MODIFIER_BITMAP[this.VK_CONTROL]: {
			GetKeyVK("F"): true,
			GetKeyVK("L"): true,
			GetKeyVK("N"): true,
			GetKeyVK("O"): true,
			GetKeyVK("P"): true
		}
	}
	)
	

Does each object need it's own key? 

Do I need to increment the object's number like 

		0: {
			this.VK_F5: true
		},

		1: {
			this.VK_F6: true
		},

**Comment by SaifAqqad** [comment]
The `0` object contains keys that need to be blocked when pressed without a modifier
```ahk
    0: {
        this.VK_F5: true
    },
```

The other objects ( in this case there's only one more) will contain the keys that need to be blocked when pressed with a modifier (The modifier is specified as the object's name)
```ahk
    this.MODIFIER_BITMAP[this.VK_CONTROL]: {
        GetKeyVK("F"): true,
        GetKeyVK("L"): true,
        GetKeyVK("N"): true,
        GetKeyVK("O"): true,
        GetKeyVK("P"): true
    }
```
So you just need to add the key's VK to the correct object inside `disabled_shortcuts`

**Comment by gittyup2018** [comment]
I don't understand, maybe I'm missing something but it looks like the EXACT same thing I posted? I don't see the difference? Please show me an example with multiple keys. VK_F5 AND VK_F6, for example

**Comment by SaifAqqad** [comment]
I've updated the comment to make it more clear

**Comment by gittyup2018** [comment]
Ahh I think I understand, thanks for clarifying. 

So to add another key VK_F6 without a modifier I just add that to the 0 object like

0: {
            this.VK_F5: true,this.VK_VKF6: true
		},
?
That seems easy enough and did compile without any errors..I'm not sure how to test/debug it though

**Comment by gittyup2018** [comment]
Thanks! that seemed to work and was so easy..one more question Is it possible to disable the modifier keys without an additional key? so I can have the alt (vk_menu) and control keys themselves disabled/ignored and handle them in AHK itself? 

I tried adding the VK_MENU to the 0 object 

0: {
            this.VK_F5: true,this.VK_MENU: true

		},

but in javascript it still seems to catch the alt/vk menu keycode.

I figured it out myself and just had to remove this.VK_MENU: 1<<2} from the MODIFIER_BITMAP object so closing this, thanks again for the quick replies!
